astro/src/content/blog/fusionauth-web-summit.mdx

```diff
@@ -405,7 +405,12 @@ Please come and say hi to FusionAuth at our booth, or see me (Brendan) whilst I
 
 FusionAuth adds login, registration, SSO, MFA, and a bazillion other features to your app in days - not months.
 
-[Click here](https://klever.kraftful.com/surveys-public/startups) to find out more and sign up to the waitlist for our exclusive program for early stage startups.
+[Click here](https://fusionauth.link/startups) to find out more and sign up to the waitlist for our exclusive program for early stage startups.
+
+**About the Author**
```

Comment:
Please remove the about the auth stuff. We never put that at the bottom of any posts.